dataactbroker/routeUtils.py

```diff
@@ -1,37 +1,20 @@
-from flask import session
 from dataactcore.utils.responseException import ResponseException
 from dataactcore.utils.jsonResponse import JsonResponse
 from dataactcore.utils.statusCode import StatusCode
 from dataactbroker.handlers.interfaceHolder import InterfaceHolder
-from dataactbroker.handlers.aws.session import LoginSession
 
 class RouteUtils:
     """ Holds utility functions for routes """
     SYSTEM_EMAIL = None
     CREATE_CREDENTIALS = None
 
     @staticmethod
-    def run_instance_function(accountManager, accountFunction, getSystemEmail = False, getSession = False, getUser = False, getCredentials = False, addTrueFlag = False):
+    def run_instance_function(accountManager, accountFunction, *functionArgs): #getSystemEmail = False, getSession = False, getUser = False, getCredentials = False, addTrueFlag = False):
```

Comment:
love this cleanup...do we need the old params for future reference, or can we remove those?